### PR TITLE
feat(docs): add ASI Alliance wallet APIs in docs

### PR DESCRIPTION
**Pull request by FelixNicolaeBucsa**
## Proposed Changes

_Adds Fetch Wallet APIs in docs_

## Linked Issues

_[if applicable, add links to issues resolved by this PR]_

## Types of changes

_What type of change does this pull request make (put an `x` in the boxes that apply)?_

- [x] Content update.
- [ ] Bug fix (non-breaking change that fixes an issue).
- [ ] New feature added (non-breaking change that adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to stop working as expected).
- [ ] Something else (e.g., tests, scripts, example, deployment, infrastructure).

## Checklist

_Put an `x` in the boxes that apply:_

- [x] I have read the [CONTRIBUTING](https://github.com/fetchai/docs/blob/main/CONTRIBUTING.md) guide
- [x] Checks and tests pass locally

## Further comments

_[if this is a relatively large or complex change, kick off a discussion by explaining why you chose the solution you did, what alternatives you considered, etc...]_

### CODE REVIEW

**Review by gautamgambhir97 — APPROVED** [review]

File: pages/apis.mdx [hunk 1]
```diff
@@ -61,4 +61,20 @@ import { GuideBox } from "../components/feature-guide-tabs";
...
 </Row>
+
+## Fetch Wallet APIs
+
+<div className="nx-text-fetch-content"></div>
+
+<Row>
+    <GuideBox
+        content={{
+            title: "Fetch Wallet APIs",
```
Comment (sh-wallet) [inline pages/apis.mdx]: It should be `ASI Alliance Wallet APIs`

File: pages/apis.mdx [hunk 2]
```diff
@@ -61,4 +61,20 @@ import { GuideBox } from "../components/feature-guide-tabs";
...
+            description: "Start working with the Fetch Wallet APIs.",
```
Comment (sh-wallet) [inline pages/apis.mdx]: All `Fetch Wallet` references should be renamed as `ASI Alliance Wallet`

File: pages/apis/asi-alliance-wallet/fetch-wallet.mdx [hunk 3]
```diff
@@ -0,0 +1,1958 @@
+import { Row, Col } from "components/mdx";
+import {
+  ApiEndpointRequestResponse,
+} from "components/api-endpoint";
+
+# ASI Alliance Wallet API
+
+<div className="nx-text-fetch-content">
+
+This comprehensive resource serves as a detailed reference guide for the **ASI Alliance Wallet APIs**, providing developers with the essential information to integrate and interact with the Fetch Browser Wallet in their decentralized applications (dApps).
```
Comment (sh-wallet) [inline pages/apis/asi-alliance-wallet/fetch-wallet.mdx]: **Fetch Browser Wallet** renamed as **ASI Browser Wallet**

File: pages/apis/asi-alliance-wallet/fetch-wallet.mdx [hunk 4]
```diff
@@ -0,0 +1,1958 @@
...
+
+</Row>
+
+## Legacy Keplr APIs (will be deprecated)
+
+<Row>
+    <Col>
+    We recognize that Keplr APIs are broadly utilized in most Cosmos dApps.
+
+    To assist dApp creators, we support Keplr APIs, allowing them to use their existing logic with the ASI Alliance Wallet. This support facilitates a gradual transition to more advanced APIs of the fetch browser wallet.
```
Comment (sh-wallet) [inline pages/apis/asi-alliance-wallet/fetch-wallet.mdx]: fetch browser wallet renamed as ASI browser wallet

**Pull request merged**